README.md

```diff
@@ -0,0 +1,67 @@
+<!-- <h1 align="center">The ARINC 853 "CSMIM" Knowledge Base</h1> -->
+<p align="center">
+  <a href="https://github.com/ARINC-IA/CSMIM">
+    <img alt="CSMIM" title="CSMIM" src="docs/csmim.png" width="300">
+  </a>
+</p>
+
+Welcome to the **ARINC 853 Knowledge Base** for the "Cabin Secure Media-Independent Messaging" Protocol.
+This repository contains standardized interface descriptions for aircraft cabin equipment.
+Here, the aerospace industry collaborates to shape the cabin network of the future.
+
+
+## Introduction
+
+This repository complements the ARINC 853 "CSMIM" standard. ARINC 853 specifies a modern
```

Comment:
maybe link to https://aviation-ia.sae-itc.com/standards/arinc853-853-cabin-secure-media-independent-messaging-csmim-protocol